context/context.go

```diff
@@ -144,8 +139,16 @@ func InitSmfContext(config *factory.Config) *SMFContext {
 		smfContext.Name = configuration.SmfName
 	}
 
-	// copy static UE IP Addr config
-	smfContext.StaticIpInfo = &configuration.StaticIpInfo
+	if env := os.Getenv("PFCP_PORT_UPF"); env != "" {
+		if v, err := strconv.Atoi(env); err == nil && v >= 0 && v <= 65535 {
+			DefaultPfcpPort = uint16(v)
+			logger.CtxLog.Infof("using PFCP_PORT_UPF from environment variables: %d", DefaultPfcpPort)
+		} else {
+			logger.CtxLog.Warnf("invalid PFCP_PORT_UPF value %q: %v. Using default value: %d", env, err, DefaultPfcpPort)
+		}
+	}
```

Comment:
I see that in some places we are using `DefaultPfcpPort` and in other cases `factory.DEFAULT_PFCP_PORT` is used. I think only one of them should be used across the codebase. Otherwise, discrepancies will arise.
For example, in this same file, `DefaultPfcpPort = uint16(v)` is overwritten and `pfcp.Port = factory.DEFAULT_PFCP_PORT` is used in another section. Should not `pfcp.Port = DefaultPfcpPort`? That is, use `DefaultPfcpPort` across the codebase?